app/models/account.rb

```diff
@@ -12,6 +12,7 @@ class Account < ApplicationRecord
 
   has_one :profile, dependent: :destroy
   has_many :phones, dependent: :destroy
+  has_many :labels
```

Comment:
we can make dependent: :destroy

Reply:
@mod This is done on db level by foreign key constraint `on_delete: :cascade`. No need to call AR callbacks (`dependent: :destroy`)